managedkafka/examples/src/main/java/examples/CreateCluster.java

```diff
@@ -64,17 +72,59 @@ public static void createCluster(
             .setRebalanceConfig(rebalanceConfig)
             .build();
 
-    try (ManagedKafkaClient managedKafkaClient = ManagedKafkaClient.create()) {
+    // Create the settings to configure the timeout for polling operations
+    ManagedKafkaSettings.Builder settingsBuilder = ManagedKafkaSettings.newBuilder();
+    TimedRetryAlgorithm timedRetryAlgorithm = OperationTimedPollAlgorithm.create(
+        RetrySettings.newBuilder()
+            .setInitialRetryDelayDuration(Duration.ofSeconds(5L))
+            .setRetryDelayMultiplier(1.5)
+            .setMaxRetryDelayDuration(Duration.ofSeconds(30L))
+            .setInitialRpcTimeoutDuration(Duration.ZERO) // ignored
+            .setRpcTimeoutMultiplier(1.0) // ignored
+            .setMaxRpcTimeoutDuration(Duration.ZERO) // ignored
+            .setTotalTimeoutDuration(Duration.ofHours(1L)) // set polling timeout to 1 hour
+            .build());
```

Comment:
Can we get the default settings as a builder + only change the value we need? Here and elsewhere. Otherwise it makes the samples very verbose.

Reply:
I'm not quite sure I follow what you mean by "get the default settings as a builder"? I'm already doing that by creating the builder and I'm only setting the `createClusterOperationSettings` which is what we need to change?

I intentionally left the RPC timeout values with the ignored comment so users know that these settings will be ignored for LROs (I should probably state explicitly that it is ignored for LROs, oops). I can remove these if you want though but left them in there for now.

Reply:
We should only set RetrySettings.Builder values where we need to change the default value.

See docs:
https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.retrying.RetrySettings.Builder

1. `setInitialRetryDelayDuration` defaults to 5 seconds, so there is no need to set it. 
2.  `setRetryDelayMultiplier` This is the default value, there is no need to set it.
3. For `setMaxRetryDelayDuration` the default is (45 seconds), so it's fine to set to 30 seconds, but might not really be necessary.
4. `setInitialRpcTimeoutDuration`, `setRpcTimeoutMultiplier`, and `setMaxRpcTimeoutDuration` are not necessary since they are ignored for LROs and you are just setting them to the default values.
I think we should only set RetrySettings.newBuilder()..setTotalTimeoutDuration(Duration.ofHours(1L)).build());

Reply:
Ahhh I see what you meant - I was looking at the defaults set in the `RetrySettings` builder (which doesn't have the LRO defaults) but the LRO-specific values are set in our `ManagedKafkaStubSettings` builder.

Fixed!